sample/HttpTrigger-Scrape-Promise/index.js

```diff
@@ -0,0 +1,26 @@
+﻿var http = require('http');
+
+module.exports = function (context, req) {
+    // Choose the host from either the body or query string
+    var host = req.body.host || req.query.host;
```

Comment:
Does this function forward an arbitrary URL, or just a host?

Perhaps you could do something like take an array of stock names, and return their prices, using `https://www.google.com/finance/info?q=MSFT,GOGL`, and parse/simplify the response to just name/price?